tests/dashbio_demos/app_clustergram.py

```diff
@@ -637,7 +659,7 @@ def display_clustergram(
                 }
             )
         if fig_opts['cluster'] is None:
-            return html.Div(
+            wrapper_content = html.Div(
                 'No clustering dimension has been selected to display. Please \
```

Comment:
Isn't it "no dimension has been selected to perform clustering along?"

Reply:
Yep, that sounds clearer.